core-bundle/src/Image/Studio/ImageResult.php

```diff
@@ -100,10 +101,18 @@ public function getImg(): array
     }
 
     /**
-     * Returns the "src" attribute of the image.
+     * Returns the "src" attribute of the image. This will return an URL by
+     * default. Set $asPath to true to get a relative file path instead.
      */
-    public function getImageSrc(): string
+    public function getImageSrc(bool $asPath = false): string
     {
+        if ($asPath) {
+            /** @var Image $image */
+            $image = $this->getPicture()->getImg()['src'];
```

Comment:
In line 116, you return `$this->getImg()['src']` but here you return `$this->getPicture()->getImg()['src']`. Intended?

Reply:
Yes, `$this->getImg()` calls `PictureInterface#getImg()` with the current project dir and static url whereas in this line we're using no arguments (and are then getting `Image`s back instead of urls).